swift/Workflow/Tests/WorkflowNodeTests.swift

```diff
@@ -179,6 +179,8 @@ final class WorkflowNodeTests: XCTestCase {
             struct State {}
 
             typealias Output = Int
+            
+            typealias Rendering = Void
```

Comment:
It appears that the Swift 5 compiler (even when running in 4.2 mode) has different behavior when it comes to type inference.